pkg/apis/api/v1alpha1/astarte_types.go

```diff
@@ -242,6 +236,37 @@ type AstarteRabbitMQSpec struct {
 	EventsExchangeName string `json:"eventsExchangeName,omitempty"`
 }
 
+type AstarteCassandraSSLConfigurationSpec struct {
+	Enabled bool `json:"enabled"`
```

Comment:
You most likely want to make this optional and with `omitempty`, given you aren't (correctly) using a pointer for it.